src/delivery-options/config/configBus.js

```diff
@@ -390,15 +390,16 @@ export const createConfigBus = (eventCallee = null) => {
        * @param {MyParcel.CarrierName|Number} obj.value - New carrier or pickup location id.
        */
       updateCurrentCarrier({ name, value }) {
-        switch (name) {
-          case FORM.CARRIER:
-            this.currentCarrier = value;
-            break;
-          case FORM.PICKUP_LOCATION:
-            if (this.pickupLocations.hasOwnProperty(value)) {
-              this.currentCarrier = this.pickupLocations[value].carrier;
-            }
-            break;
+        if (FORM.CARRIER === name) {
+          this.currentCarrier = value;
+        } else if (FORM.PICKUP_LOCATION === name) {
+          const foundPickupLocation = this.pickupLocations.find((pickupLocation) => {
```

Comment:
Je kunt hier `location_code` destructuren en dan kan de `return` er uit en kan het een oneliner worden.

Reply:
Dat heb ik dus expres niet gedaan om `ESLint: Identifier 'location_code' is not in camel case.(camelcase)` te vermijden 😢

Reply:
Je zou die kunnen aliassen 

`
this.pickupLocations.find((({location_code : locationCode}) => locationCode  === value));
`

Reply:
dan is de regel weer te lang :)))))))

Reply:
Ah, prima dan zo